<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="ButtonUp">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>470</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonLeft">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>520</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonDown">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>570</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonRight">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>520</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonAction">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>500</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonPause">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>470</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonStart">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>620</y>
      <width>75</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonEnd">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>620</y>
      <width>75</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="matrix">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>400</width>
      <height>420</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelAction">
    <property name="geometry">
     <rect>
      <x>278</x>
      <y>560</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Black</family>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ACTION</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="LabelPause">
    <property name="geometry">
     <rect>
      <x>348</x>
      <y>530</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Black</family>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>PAUSE</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="LabelQuit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>640</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Black</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>QUIT</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="LabelStart">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>640</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Black</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>START</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="LabelStatus">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>180</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Black</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="LabelGame">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>13</y>
      <width>181</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Black</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
